docs_zh-CN/demo.md

```diff
@@ -1 +1,85 @@
-# 演示
\ No newline at end of file
+# 演示
+
+我们提供了多模态/单模态（基于激光雷达/视觉）、室内/室外场景的3D检测和3D语义分割演示的脚本，预训练模型可以从 [model zoo](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/model_zoo.md/) 下载。我们也提供了 KITTI、SUN RGB-D、nuScenes 和 ScanNet 数据集的预处理样本数据，你可以根据我们的预处理步骤使用任何其它数据。
```

Comment:
Maybe add space between English word `3D` and the surrounding Chinese words?

Reply:
Problems solved now.